packages/openapi-react-query/test/index.test.tsx

```diff
@@ -0,0 +1,290 @@
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+import { server, baseUrl, useMockRequestHandler } from "./fixtures/mock-server.js";
+import type { paths } from "./fixtures/api.js";
+import createClient from "../src/index.js";
+import createFetchClient from "openapi-fetch";
+import { render, renderHook, screen, waitFor } from "@testing-library/react";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import React, { Suspense, type ReactNode } from "react";
```

Comment:
ignorable suggestion:

```suggestion
import { Suspense, type ReactNode } from "react";
```

IIRC the latest recommendation is to _not_ import the `React` global to support future upcoming versions. Even though this library doesn’t use React internals (it makes it much more annoying to ship if it does), we should still test that these tests work without the React global (which means proper setting of `"jsx"` in `tsconfig.json`, etc.).

This isn’t imperative before merging, but can just save a teensy bit of headache for React 19.

Reply:
I wrongly configured the tsconfig jsx. It is now on `react-jsx` and React does not have to be imported anymore. Tests are green.